javers-spring-boot-starter-mongo/build.gradle

```diff
@@ -2,9 +2,9 @@ dependencies {
     compile project(':javers-spring')
     compile project(':javers-persistence-mongo')
     //should be on application's classpath when using JaversSpringBootStarterMongo
-    provided 'org.springframework.boot:spring-boot-starter-data-mongodb:1.3.0.RELEASE'
```

Comment:
Optional to mniej wiecej to samo tylko nie wrzucasz klientowi konkretnej wersjii, do przegadania...